integration_test/third_party_apps_data/applications/aerospike/metadata.yaml

```diff
@@ -69,6 +69,7 @@ expected_metrics:
       namespace_name: .*
     monitored_resource: gce_instance
     type: workload.googleapis.com/aerospike.namespace.scan.count
+    optional: true
```

Comment:
This aerospike metric is broken for aerospike 6.X - see issue raised https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21834

Reply:
This should also be added as a comment in the metadata file until it is resolved.